### PR TITLE
fix/terminal survey select UI

### PR DESCRIPTION
**Pull request by Skisocks**
PR to fix the issue with the jx-pipline cli where on some terminals a new selection screen would be printed every time the selection was updated.
Required updating survey package to v2 and some minor refactors to integrate.

### CODE REVIEW

**Comment by Skisocks** [comment]
Merged the Add_updatebot PR into this branch

File: .jx/updatebot.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+              - jenkins-x-plugins
+            repositories:
+              include:
+                - jx-project
+                - jx-pipeline
+            package: github.com/jenkins-x/jx-helpers
+            upgradePackages:
+              include:
+                - "github.com/jenkins-x/*"
+                - "github.com/jenkins-x-plugins/*"
```
Comment (ankitm123) [inline .jx/updatebot.yaml:22]: NIT: EOL

**Review by ankitm123 — COMMENTED** [review]
minor comment, else good to merge.

**Pull request merged**